lib/openstudio/workflow/jobs/run_energyplus.rb

```diff
@@ -52,13 +54,14 @@ def perform
 
     # Checks and configuration
     raise 'No run_dir specified in the registry' unless @registry[:run_dir]
-    ep_path = @options[:energyplus_path] ? @options[:energyplus_path] : nil
+
+    ep_path = @options[:energyplus_path] || nil
     @logger.warn "Using EnergyPlus path specified in options #{ep_path}" if ep_path
 
     @logger.info 'Starting the EnergyPlus simulation'
-    @registry[:time_logger].start('Running EnergyPlus') if @registry[:time_logger]
+    @registry[:time_logger]&.start('Running EnergyPlus')
```

Comment:
TIL (today I learned) about the Safe Navigation Operator (&.) in Ruby.

Reply:
my favorite... it was after ruby 2.2 though.